docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,546 @@
+While MUI is compatible with any styling solution (as long as the styles have more specificity, e.g. Tailwind CSS), many developers still felt the need to learn something new: the [`makeStyles`](/styles/basics/#hook-api) API.
+
+2. Our React integration with JSS (`@mui/styles`) is **too slow** to unlock the next layer of customization DX we aim for.
+   The static CSS generation using v4 was fast enough, even [faster](https://codesandbox.io/s/nb05w?file=/src/App.js) than emotion.
+   However, the dynamic style generation was too slow to be used in production. We would have needed to reimplement it.
+3. Many developers were advocating for MUI to [migrate to styled-components](https://github.com/mui-org/material-ui/issues/6115).
+   It would mean giving up on the custom React JSS wrapper we maintain.
+   From our experience, maintaining a custom styling solution takes a considerable amount of time. Is it the best use of our time?
+
+After [exploring](https://github.com/mui-org/material-ui/issues/22342) many different options.
+We settled on what we believe is a great tradeoff to **solve** the above issues:
+
+1. We have made `styled` the lowest level primitive to add styles.
+   This API is already known by many.
+2. We have defined a common interface with concrete implementations:
+
+   - `@mui/styled-engine`: implemented with emotion (default).
+   - `@mui/styled-engine-sc`: implemented with styled-components
+   - If you are using a different styled engine, feel free to contribute a wrapper. For instance, there is [one attempt with goober](https://github.com/mui-org/material-ui/pull/27776), a library obsessing on bundle size (3kB gzipped).
+
+   This allows developers to swap between different styled engines. For instance, styled-components users do no longer need to bundle emotion **and** styled-component, nor do they need to configure the server-side rendering for each.
+   How does the [swap works](/guides/styled-engine/#how-to-switch-to-styled-components)? The same way it does from React to Preact.
+
+3. We have started to [sponsor](https://opencollective.com/emotion) emotion with a $1,000/month grant. It's in our best interest to help ensure the library keeps pushing the envelope, staying state of the art in a competitive space.
+
+The first immediate benefit was at the **performance** level. the `<Box>` component is [x5-x10 more performant](https://codesandbox.io/s/zlh5w?file=/src/App.js) in v5, compared to v4.
+
+We would like to thanks all the community contributors that made the migration of the components and documentation possible in [#24405](https://github.com/mui-org/material-ui/issues/24405) and [#16947](https://github.com/mui-org/material-ui/issues/16947) @natac13, @vicasas, @mngu, @kodai3, @xs9627, @povilass, @duganbrett, @vinyldarkscratch, and more.
```

Comment:
Should names be links to the Github profiles? If not, I'd skip the "@"

Reply:
I have added the links